### PR TITLE
[PAY-1632] Clean up and improve performance of music confetti

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Switch svg to inlined Path2D for perf improvements, Bunch of code reduction

Currently the diff in `ConnectedMusicConfetti` is there for testing purposes (trigger confetti pressing `x`). Will remove that before merge.

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

To test, visit https://demo.audius.co/rj-pay-1632/feed and press `x`

* chrome
* firefox
* safari
* edge

Also tested matrix mode

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-pay-1632

File: packages/web/src/components/background-animations/MusicConfetti.tsx [hunk 1]
```diff
@@ -1,81 +1,34 @@
...
-    gravity,
-    rotate,
-    swing,
-    particleRate,
-    onCompletion
-  )
-  confetti.run()
-  return confetti
-}
+const COLORS = ['#cc0fe0', '#7e1bcc']
```
Comment (schottra) [inline packages/web/src/components/background-animations/MusicConfetti.tsx:24]: Just wondering if we should define these as constants somewhere in themes? Are they from the color palette?

File: packages/web/src/components/background-animations/MusicConfetti.tsx [hunk 2]
```diff
@@ -84,65 +37,41 @@ type MusicConfettiProps = {
...
-        isMatrix
+        isMatrix ? undefined : PATHS,
+        isMatrix ? undefined : COLORS,
+        recycle || false,
+        limit || 200,
+        friction || 0.99,
+        gravity || 0.2,
+        isMatrix ? 0 : rotate || 0.1,
+        isMatrix ? 0 : swing || 0.01,
+        particleRate || 0.1,
```
Comment (schottra) [inline packages/web/src/components/background-animations/MusicConfetti.tsx:70]: Will also advocate for magic numbers to move to constants at the top if they have semantic meaning and might need to be tweakable.

File: packages/web/src/components/background-animations/MusicConfetti.tsx [hunk 3]
```diff
@@ -157,46 +86,20 @@ const UnconditionalMusicConfetti = (props: Required<MusicConfettiProps>) => {
...
-      {...sizing}
+      width={window.innerWidth}
+      height={window.innerHeight}
       style={{
         pointerEvents: 'none',
         position: 'absolute',
         top: 0,
         left: 0,
-        zIndex: props.zIndex,
+        zIndex: zIndex || 16,
```
Comment (schottra) [inline packages/web/src/components/background-animations/MusicConfetti.tsx:99]: Is 16 arbitrary or is it meant to be some other item's zIndex +1? We have a[ zIndex file](https://github.com/AudiusProject/audius-client/blob/f94ebec5d626d8fcd4e376345b32dd690b710e22/packages/web/src/utils/zIndex.ts#L5-L6) that you could pull a constant from and reference here if we wants this to always live above certain elements

**Review by schottra — APPROVED** [review]
_So much better_ in the demo. Good catch and thank you for fixing this!

File: packages/web/src/components/background-animations/MusicConfetti.tsx (hunk 3 above)
Comment (raymondjacobson) [reply packages/web/src/components/background-animations/MusicConfetti.tsx:99]: 👍

File: packages/web/src/components/background-animations/MusicConfetti.tsx (hunk 1 above)
Comment (raymondjacobson) [reply packages/web/src/components/background-animations/MusicConfetti.tsx:24]: nice -- was able to pull them from the theme.

**Comment by raymondjacobson** [comment]
@schottra unf our hotkey util doesn't support multiple modifiers + multiple keys right now, so going to punt on that. but left the hotkey hook in there commented out for future iteration

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-pay-1632

**Pull request merged**